portal-impl/src/com/liferay/portal/service/impl/CountryLocalServiceImpl.java

```diff
@@ -14,11 +14,253 @@
 
 package com.liferay.portal.service.impl;
 
+import com.liferay.portal.kernel.dao.orm.QueryUtil;
+import com.liferay.portal.kernel.exception.CountryA2Exception;
+import com.liferay.portal.kernel.exception.CountryA3Exception;
+import com.liferay.portal.kernel.exception.CountryIddException;
+import com.liferay.portal.kernel.exception.CountryNumberException;
+import com.liferay.portal.kernel.exception.DuplicateCountryException;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.model.Country;
+import com.liferay.portal.kernel.model.Organization;
+import com.liferay.portal.kernel.model.OrganizationConstants;
+import com.liferay.portal.kernel.model.SystemEventConstants;
+import com.liferay.portal.kernel.model.User;
+import com.liferay.portal.kernel.service.ServiceContext;
+import com.liferay.portal.kernel.systemevent.SystemEvent;
+import com.liferay.portal.kernel.util.LocaleUtil;
+import com.liferay.portal.kernel.util.OrderByComparator;
+import com.liferay.portal.kernel.util.Validator;
 import com.liferay.portal.service.base.CountryLocalServiceBaseImpl;
 
+import java.util.List;
+import java.util.Map;
+
 /**
  * @author Brian Wing Shun Chan
  * @see CountryLocalServiceBaseImpl
  */
 public class CountryLocalServiceImpl extends CountryLocalServiceBaseImpl {
+
+	@Override
+	public Country addCountry(
+			boolean active, String a2, String a3, boolean billingAllowed,
```

Comment:
The parameter order here should match the service.xml file. So `active` should come after `a2` and `a3`.